MMA Removal Utility/README.md

```diff
@@ -0,0 +1,91 @@
+----------------------------------------------
+
+> The Azure Tenant Security Solution (AzTS) was created by the Core Services Engineering & Operations (CSEO) division at Microsoft, to help accelerate Microsoft IT's adoption of Azure. We have shared AzTS and its documentation with the community to provide guidance for rapidly scanning, deploying and operationalizing cloud resources, across the different stages of DevOps, while maintaining controls on security and governance.
+<br>AzTS is not an official Microsoft product – rather an attempt to share Microsoft CSEO's best practices with the community.
+
+# AzTS MMA Removal Utility
+
+-----------------------------------------------------------------
+## Overview 
+
+AzTS MMA Removal utility can be used to remove MMA extension/agent from Azure Virtual Machine (VMs) across subscriptions. Microsoft Monitoring Agent (MMA) will be retired on 31 August 2024 and users need to migarte to Azure Monitor agent (AMA). If you have already migrated all of your Virtual Machines (VMs) to start using AMA for monitoring, it is recommened to remove MMA agent to avoide dupliaction of logs.
```

Comment:
If you have already **migrated** all of your Virtual Machines (VMs) to start using AMA for monitoring -> If you have already **configured** all of your Virtual Machines (VMs) to start using AMA for monitoring.

Configured seems to be better word in current context.